tests/unit/conftest.py

```diff
@@ -6,9 +6,7 @@
 
 import dataactcore.config
 # Load all models so we can access them through baseModel.Base.metadata
-from dataactcore.models import (    # noqa
```

Comment:
If this isn't needed anymore, we should probably remove the comment immediately proceeding.